src/main/java/org/ow2/proactive/catalog/dto/CatalogObjectMetadata.java

```diff
@@ -57,7 +57,7 @@
     protected final String kind;
 
     @JsonProperty("bucket_id")
-    protected final Long bucketId;
+    protected final String bucketId;
 
```

Comment:
what is the specific reason to change the bucketId to a String but not keep it as a long?